README.md

```diff
@@ -87,9 +89,11 @@ configuration options [here](docs/options.md). If you are on Windows, check [her
 Install the IronFunctions CLI tool:
 
 ```sh
-curl -sSL http://get.iron.io/fn | sh
+curl -L http://get.iron.io/fn | sh
```

Comment:
My bad.  I saw "-sSL" and immediately thought secure sockets layer, so I typed "--ssl".